prov/gni/src/gnix_sep.c

```diff
@@ -119,6 +119,11 @@ static int gnix_sep_tx_ctx(struct fid_ep *sep, int index,
 		(index >= sep_priv->info->ep_attr->tx_ctx_cnt))
 		return -FI_EINVAL;
 
+	if (attr && attr->op_flags & ~GNIX_EP_OP_FLAGS) {
```

Comment:
Sorry, my bad.  I was confused.  I think you're right that the ops flags check is okay as is.  The caps need to be check for subset.  It does say this in the separate sections for fi_tx_context and fi_rx_context in the man pages.